src/AutoBuild.jl

```diff
@@ -713,8 +722,10 @@ function autobuild(dir::AbstractString,
             default_host_platform;
             verbose=verbose,
         )
-        host_artifact_paths = setup_dependencies(prefix, Pkg.Types.PackageSpec[getpkg(d) for d in dependencies if is_host_dependency(d)], default_host_platform; verbose=verbose)
```

Comment:
I'm doing this in a separate PR (#1129), where I'm also going to add tests for that

Reply:
#1129 merged and this PR rebased on master